src/pages/CreateAccount.js

```diff
@@ -147,21 +181,26 @@ export default function CreateAccount({
             />
           </FormSection>
           <FormSection>
-            <Checkbox
-              label={
-                <>
-                  By checking this box you agree to our{" "}
-                  <a
-                    className="bold underlined-link"
-                    href="https://raw.githubusercontent.com/zeeguu/browser-extension/main/PRIVACY.md"
-                    target={isMobile() ? "_self" : "_blank"}
-                    rel="noreferrer"
-                  >
-                    {strings.privacyNotice}
-                  </a>
-                </>
-              }
-            />
+            <sC.CheckboxWrapper>
+              <input
+                onChange={handleCheckPrivacyNote}
+                checked={checkPrivacyNote}
+                id="checkbox"
+                name=""
+                value=""
+                type="checkbox"
+              ></input>
+              <label>
```

Comment:
Btw, here I am not using your component to avoid the situation where the user clicks on the link and accepts the policy without noticing. So here we have to have the user explicitly click the square.